src/main/java/seedu/address/logic/commands/DeletePatient.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+public class DeletePatient extends DeleteCommand {
```

Comment:
Again, Add "Command" behind the name of your class, and don't extend the DeleteCommand (it's a pseudonym for DeletePersonCommand if you have not read the body of the code; it's not a generic delete command class)